### DISCUSSION TITLE
fix error [!] Unable to find the Xcode project `/Users/name/Desktop/wallet-core-master/swift/TrustWalletCore.xcodeproj` for the target `Pods-WalletCore`.

### DISCUSSION THREAD

**Discussion by nardellascuola** [trustwallet/wallet-core#1777] [Q&A]
how to fix error [!] Unable to find the Xcode project `/Users/xxx/Desktop/wallet-core-master/swift/TrustWalletCore.xcodeproj` for the target `Pods-WalletCore`. after pod install

**Comment by hewigovens** [comment]
Have you run bootstrap.sh successfully?

**Reply by nardellascuola** [reply]
yes but it is another error: : line 7: tools/install-dependencies: No such file or directory

**Reply by hewigovens** [reply]
please read this https://developer.trustwallet.com/wallet-core/developing-the-library/building?
